dalite_xblock/public/js/dalite_xblock.js

```diff
@@ -4,8 +4,97 @@ function DaliteXBlock(runtime, element) {
     // hack to make LTI Consumer css applied to Dalite XBlock
     $(element).addClass("xblock-student_view-lti_consumer xblock-student_view");
     $(element).children(".xblock-dalite").addClass("lti_consumer");
-    
+
+    $(function ($) {
```

Comment:
@jbzdak Yeah, I hear you...  I can't think of a better way to patch in the change either, but am still interested to hear from Braden on whether this is worth an upstream PR.

I'll do a full test of your latest changes before your start of day on Monday (am away from my devstack right now), but it's looking good so far. 😄